plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/transactionlog/DeltaLakeSchemaSupport.java

```diff
@@ -686,6 +702,7 @@ private static Type buildType(TypeManager typeManager, JsonNode typeNode, boolea
             // For more info, see https://delta-users.slack.com/archives/GKTUWT03T/p1585760533005400
             // and https://cwiki.apache.org/confluence/display/Hive/Different+TIMESTAMP+types
             case "timestamp" -> TIMESTAMP_TZ_MILLIS;
+            case "variant" -> throw new UnsupportedTypeException("variant");
```

Comment:
This should be trino exception

Reply:
Why? This exception shouldn't reach users.

Reply:
right

(This might be the case where checked exception would actually fits better)